### PR TITLE
cleanup return if appliedmainfestWork is not found

### PR DESCRIPTION
**Pull request by zhiweiyin318**
Signed-off-by: Zhiwei Yin <zyin@redhat.com>

### CODE REVIEW

File: pkg/operators/klusterlet/controllers/klusterletcontroller/klusterlet_managed_reconcile.go [hunk 1]
```diff
@@ -168,6 +168,9 @@ func (r *managedReconcile) clean(ctx context.Context, klusterlet *operatorapiv1.
 func (r *managedReconcile) cleanUpAppliedManifestWorks(ctx context.Context, klusterlet *operatorapiv1.Klusterlet, config klusterletConfig) error {
 	appliedManifestWorks, err := r.managedClusterClients.appliedManifestWorkClient.List(ctx, metav1.ListOptions{})
 	if err != nil {
+		if errors.IsNotFound(err) {
```
Comment (qiujian16) [inline pkg/operators/klusterlet/controllers/klusterletcontroller/klusterlet_managed_reconcile.go]:
```go
if errors.IsNotFound(err) {
	return nil
}
if err != nil {
	return fmt.Errorf("unable to list AppliedManifestWorks: %w", err)
}
```

Comment (zhiweiyin318) [reply pkg/operators/klusterlet/controllers/klusterletcontroller/klusterlet_managed_reconcile.go]: done

**Pull request merged**